setup/cicd.rst

```diff
@@ -92,4 +93,7 @@ Adjust heap size of JRE
 
 Travis CI
 ---------
-To check pull requests before merge into master there is a Travis CI build which runs the docker-compose image.
+* To check pull requests before merge into master there is a Travis CI build which runs the docker-compose image.
+    * Travis runs unit tests, but no integration tests
+    * At the current state, Travis CI builds only mico-core, not mico-admin
```

Comment:
Travis builds also `mico-admin`. However we don't have any unit tests for `mico-admin`.